src/components/headers/MenuConsts.js

```diff
@@ -24,10 +28,38 @@ const MENU_ROUTES :Object = {
   WORKING_WITH_US: Routes.WORKING_WITH_US,
 };
 
+const PRODUCT_MENU_ITEMS :Object[] = [
+  {
+    BLURB: 'Manage a person moving through pretrial system',
+    ICON: PCMLogo,
+    NAME: 'Pretrial Case Management',
+    ROUTE: Routes.PCM,
+  },
+  {
+    BLURB: 'Collect and utilize behavioral health data',
+    ICON: CARELogo,
+    NAME: 'CARE',
+    ROUTE: Routes.CARE,
+  },
+  {
+    BLURB: 'Collaborative human centered research tool',
+    ICON: OLLogo,
```

Comment:
human-centered?

Reply:
@anzioka oops sorry, forgot to address this—I'm always hesitant to make any changes to text or images, because those things are written/created by other people and just handed off to me. But in this case, I do agree with you that it should probably be "human-centered", so I'll make the change!